### DISCUSSION TITLE
[ps tv] got a problem using ds 3 controller via ps tv

### DISCUSSION THREAD

**Discussion by bellonok** [diasurgical/DevilutionX#1607] [Q&A] [answered]
hi everybody! got a problem using ds3 controller on ps tv, the problem is l1 and r1 buttons doesnt do anything, however they supposed to be using for health and mana elixirs, I look into ".ini" file, but there was nothing like about l1 and r1 remap or thomethig else..
can someone help me please

**Comment by AJenbo** [comment]
https://github.com/diasurgical/devilutionX/wiki/Game-Controller-Scheme

you can also save the mapping in the ini

**Reply by bellonok** [reply]
I get it, but as they say l1 and r1 are uses elixirs by default, but not for me, or u talking about this (see pict.)![image](https://user-images.githubusercontent.com/82802821/115204722-7aab0680-a101-11eb-9841-9a85a39068fa.jpeg)

**Comment by AJenbo** [comment, marked as answer]
yeah remap them

**Reply by bellonok** [reply]
Got it, thnx for help👋🏼
